lib/webauthn/authenticator_assertion_response.rb

```diff
@@ -14,7 +14,7 @@ def self.from_client(response, relying_party: WebAuthn.configuration.relying_par
       encoder = relying_party.encoder
 
       user_handle =
-        if response["userHandle"]
+        if response["userHandle"] && String === response["userHandle"]
```

Comment:
What do you think about:

```suggestion
        if response["userHandle"] && response["userHandle"].is_a? String
```

IMO it feels more readable 🙂

Reply:
I am good. It's just my personal preference. I am open to embracing whatever style guide / practice team is already familiar with for this particular repo.

Reply:
Can we go with `is_a?` then? Just to be consistent with the rest of the code 🙂 Thanks!

Reply:
Sorry this slipped out of my inbox. I pushed a commit. Please check once more and let's get this merged!